http/cookie.ts

```diff
@@ -131,7 +131,7 @@ function toString(cookie: Cookie): string {
   if (typeof cookie.maxAge === "number" && Number.isInteger(cookie.maxAge)) {
     if (cookie.maxAge < 0) {
       throw new RangeError(
-        "Max-Age must be an integer superior or equal to 0. Cookie ignored.",
+        `Cookie Max-Age must >= 0: received ${cookie.maxAge}`,
```

Comment:
Can this be a bit more formal?

Reply:
Sorry for the grammar in that one. What about:
`Cannot serialize cookie as Max-Age must be >= 0: received ${cookie.maxAge}`?